src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -15,23 +16,38 @@ import * as jsJoda from '@js-joda/core';
 import '@js-joda/timezone';
 
 import { ChplTextField } from 'components/util';
+import theme from 'themes/theme';
 
 const useStyles = makeStyles({
   content: {
     margin: '16px 0',
+    display: 'flex',
+  },
+  subContent: {
```

Comment:
It's really helpful to use meaningful names. Just reading `subContent` doesn't tell me anything, especially when the `content` class itself isn't used. Is this a `dateTimeLayout` class? Or a `dataEntries` class? Something else?